labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -25,7 +25,12 @@ object OptionExercise01 {
    * - does not exist: 					"not existing"
    */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+    rooms get room match {
+      case None => "not existing"
+      case Some(None) => "empty"
+      case Some(Some("locked")) => "not available"
```

Comment:
Well when you call `get` on a map, you get an `Option[map's value]`. Since the map's value is `Option[String]`, calling `get` gives an `Option[Option[String]]`, so if you want the inside part you have to do `Some(Some(s))`